docs/source/connection_table.rst

```diff
@@ -5,9 +5,13 @@ The connection table maps out the way input/output devices are connected to each
 .. image:: img/connection_diagram.png
 	:alt: Example wiring diagram.
 
-Here we see two :py:class:`PseudoclockDevice <labscript.PseudoclockDevice>` instances in the top tier of the diagram. They do not have a parent device that tells them when to update their output (this is true for all :py:class:`PseudoclockDevice <labscript.PseudoclockDevice>` instances). However, all but one (the master pseudoclock device) must be triggered by an output clocked by the master pseudoclock device. 
+Here we see two :py:class:`PseudoclockDevice <labscript.core.PseudoclockDevice>` instances in the top tier of the diagram. They do not have a parent device that tells them when to update their output (this is true for all :py:class:`PseudoclockDevice <labscript.core.PseudoclockDevice>` instances). However, all but one (the master pseudoclock device) must be triggered by an output clocked by the master pseudoclock device. 
```

Comment:
Not that we should change literally any of this, but just wanted to point out that there is an abbreviated short-hand for these cross-references.
```python
:py:class:`PseudoclockDevice <labscript.core.PseudoclockDevice>`
```
can be written as
```python
:class:`~.PseudoclockDevice`
```
Basic summary is that `:py` namespace is presumed, leading with a `.` tells sphinx to search the namespace for the class that ends with `PseudoclockDevice`, and the `~` tells sphinx to only show the final element (so not the module paths).

Again, don't change it. Work is done so whatever. Moreover, the `~` and `.` shorthands can't be used simultaneously for intersphinx cross-references (from other repos) anyway so I may as well get used to putting in full paths regardless.

Which reminds me, going to have to play whack-a-mole with all the cross-references that are going to break everywhere else.